__tests__/endpoints.test.js

```diff
@@ -58,4 +58,41 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH", () => {
+    test('should update votes on article and return correct article.', () => {
+      return request(app)
+        .patch("/api/articles/10")
+        .send({ inc_votes: 15 })
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toMatchObject({
+            author: "rogersop",
+            title: "Seven inspirational thought leaders from Manchester UK",
+            article_id: 10,
+            body: expect.any(String),
+            topic: "mitch",
+            created_at: expect.any(String),
+            votes: 15,
+          });
+        });
+    });
+    test("should return 400 when submitting invalid key.", () => {
+      return request(app)
+        .patch("/api/articles/10")
```

Comment:
Test - Patch to an article that doesnt exist?
test - patch to invalid article id